.travis.yml

```diff
@@ -24,7 +24,7 @@ script:
   - ./build.sh
   - ./install.sh
   - ./test/validateTarFiles.sh
-  - ./test/runTest.sh -c
+  - ./test/runTest.sh -c -a allocatorApp,dataTxnApp,multiThreadedApp,slowFixDecoderApp
```

Comment:
make this the default to run test for all apps, instead of just slow fix decoder ?